tests/Basic/ErrorHandlingTest.php

```diff
@@ -9,13 +9,13 @@
  */
 final class ErrorHandlingTest extends TestCase
 {
-    const NOT_FOUND_XPATH = '//html/./invalid';
+    private const NOT_FOUND_XPATH = '//html/./invalid';
 
-    const NOT_FOUND_EXCEPTION = 'Exception';
+    private const NOT_FOUND_EXCEPTION = 'Exception';
 
-    const INVALID_EXCEPTION = 'Exception';
+    private const INVALID_EXCEPTION = 'Exception';
```

Comment:
These are technically a BC break, but I think the risk of that is very low.

If we don't want to risk it, we can use `@internal`/`@deprecated` + `public` instead.

Reply:
these constants have no reason to be reused by driver repositories, so this is fine.